src/components/CalendarCell/CalendarCell.visual.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { VisualTestContainer } from '../../../test/visual/VisualTestContainer';
+import { CalendarCell } from './';
+
+class CalendarCellVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualTestContainer>
+        <CalendarCell time="14" {...this.props} />
+      </VisualTestContainer>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
+      },
+    ],
+  },
+];
```

Comment:
In this array, you can add more visual tests.
So for instance, we would probably want a visual test for each time, timeType, and stretchable